docs/download/downloading_data.md

```diff
@@ -0,0 +1,261 @@
+# Downloading Data
+
+## Downloading data from Collaboratory repository
+
+Academic research cloud infrastructure built to house ICGC data.
+
+Repository information can be found [here](https://docs.icgc.org/download/repositories/#collaboratory)
+
+#### Prerequisites
+
+1.  [Apply for DACO access](https://github.com/icgc-dcc/dcc-docs/blob/download-doc-update/docs/download/data-access.md#apply-for-access-to-controlled-data)
+2.  Download and install score-client software [Tarball](/software/download/#score-client), [Docker](https://hub.docker.com/r/overture/score/) (Download/configuration instructions [here](https://docs.icgc.org/download/guide/#installation-of-the-score-client))
```

Comment:
like above comments, use relative URL: `[here](/download/guide/#installation-of-the-score-client)`

Reply:
Do not need to mention Docker here, it is not something user has to install. It may scare someone who never used Docker.

Reply:
corrected with relative URL and removed Docker